hooks/addon

```diff
@@ -135,7 +122,7 @@ login() {
   host_pw="$(safe get "secret/$GENESIS_VAULT_PREFIX/webui:password")"
   insecure=""
   want_feature "self-signed-cert" && insecure="-k"
-  fly -t "$host_env" login --username="$host_user" --password="$host_pw" $insecure
+  fly -t "$host_env" login --username="$host_user" --password="$host_pw" --concourse-url "$(get_host_param external_url)" $insecure
```

Comment:
You still need to run has_target to ensure we're safe -- otherwise we could overwrite an existing target with the same name -- see line 103.  If there is no target, add the --concourse-url args, otherwise leave them off (same way `$insecure` works:  `url=""; has_target || url='--concourse-url "'"$(get_host_param external_url)"'"'`

Reply:
or if you want to use less magic and more safe method:
```
if has_target ; then
  fly -t "$host_env" login --username="$host_user" --password="$host_pw" $insecure
else
  fly -t "$host_env" login --username="$host_user" --password="$host_pw" --concourse-url "$(get_host_param external_url)" $insecure 
fi
```

Reply:
it shouldn't if used in a if or conditional